functions/influxdb+grafana.bash

```diff
@@ -299,10 +306,15 @@ grafana_install(){
   if [ $FAILED -eq 2 ]; then echo -n "FAILED "; return 2; else echo -n "OK "; fi
   cond_echo ""
 
-  # password reset required if Grafana password was already set before (no first-time install)
-  echo -n "Resetting Grafana admin password... "
-  cond_redirect grafana-cli admin reset-admin-password admin || FAILED=2
-  if [ $FAILED -eq 2 ]; then echo -n "FAILED "; return 2; else echo -n "OK "; fi
+  # password reset required if Grafana password was already set before (no first-time install);
+  # do not set password for fresh installs because grafana.db will be created with root permissions
+  # which causes followup issues
+  if [ -z "$grafana_first_time" ]; then
```

Comment:
I actually just fixed #702 (which was about removing the FAILED variable)

Please rework InfluxDB/Grafana install to be consistent and fail right away, too (and remember to return, not exit !)

Reply:
I would not mix this up right now and better get this merged without further delay, as we have a broken installer right now.
My proposal: I will try the latest stuff on my test installation (new install, re-install). Then see if the bats tests work locally and then we could merge. Fixing the CI is in #917.

Reply:
ok go ahead. Let me know when you want me to merge